projects/cashmere/src/lib/sass/modal.scss

```diff
@@ -207,3 +212,10 @@ $modal-border: 1px solid $gray-200;
     justify-content: center;
 
 }
+
+@mixin hc-modal-resizable {
+    resize: both;
+    overflow: hidden;
+    min-height: 300px;
```

Comment:
This seems pretty tall for a min height. I've got a modal as short 170px in PB

Reply:
@corykon I took a look into this one and header/footer both have `min-heights` of 55px and body has a `min-height` of 120px.  So adding those up I brought the resizable min height down to 230px.